app/config.ts

```diff
@@ -47,4 +49,6 @@ export const config: Config = {
   customBrandingEnabled: Boolean(
     process.env.CUSTOM_BRANDING_ENABLED === 'true'
   ),
+  noticeEnabled: Boolean(process.env.NOTICE_ENABLED === 'true'),
+  noticeText: process.env.NOTICE_TEXT || '',
```

Comment:
Can probably add the default value here pretty safely. 
```suggestion
  noticeText: process.env.NOTICE_TEXT || 'This is a live demo of <a href="https://www.verified.inc/solutions/for-people#1-click-sign-up">1-Click Signup'</a>. You will see dummy data. To try 1-Click Signup with real data, try our <a href=https://1click.demo.verifiedinc.com/>Production demo</a>
```

Something like that

Reply:
Thanks for the suggestion!